shotover/src/observability/mod.rs

```diff
@@ -86,7 +86,7 @@ impl LogFilterHttpExporter {
                                                 format!("{:?}", error),
                                             )
                                         }
-                                        Ok(()) => rsp(StatusCode::NO_CONTENT, Body::empty()),
+                                        Ok(()) => rsp(StatusCode::OK, Body::empty()),
```

Comment:
what does this do?

Reply:
I'll revert this, turns out NO_CONTENT was correct.